cpp/include/cudf/stream_compaction.hpp

```diff
@@ -264,7 +259,7 @@ std::unique_ptr<table> drop_duplicates(
  *
  * @return Table with unique rows in an unspecified order.
```

Comment:
Avoid "unique" in `distinct` definition:
```suggestion
 * @return Table with distinct rows in an unspecified order.
```
I see what you're aiming for with "unique rows" (`distinct`) vs. "unique rows from each sequence of equivalent rows" (`unique`), but I don't find it quite clear enough. We could declare something like the message below and then always use the word "unique" to mean "no adjacent equivalent" and "distinct" to mean "no other equivalent". A shortened form of this could be used in the body of the docstrings to provide sufficient definitions, and then we could shorten all the `@param`/`@return` declarations to use that definition rather than spell it all out.

> A row is distinct if there are no equivalent rows in the table. A row is unique if there is no adjacent equivalent row. That is, keeping distinct rows removes all duplicates in the table/column, while keeping unique rows only removes duplicates from consecutive groupings.